### PR TITLE
Reword the Hetzner API's aggregation of some API responses

### PR DESCRIPTION
**Pull request by fdr**
`#filter_map` is makes this a bit more legible.  But just a bit.

### CODE REVIEW

**Comment by fdr** [comment]
I think we can do better to optimize out the structural similarity...please hold

**Comment by fdr** [comment]
Okay nevermind, I have a different approach, I can't find a greatly more satisfying rewording.

File: lib/hosting/hetzner_apis.rb [hunk 1]
```diff
@@ -59,30 +59,29 @@ def process_items(items, failovers_map)
...
+      mask = subnet["ip"].include?(":") ? 64 : subnet.fetch("mask")
+
+      {
+        ip_address: "#{subnet["ip"]}/#{mask}",
+        source_host_ip: subnet["server_ip"],
+        is_failover: subnet["failover_ip"]
+      }
     end
 
     matching_ips
```
Comment (furkansahin) [inline lib/hosting/hetzner_apis.rb:87]: this line is redundant now

**Comment by fdr** [comment]
@furkansahin alright I think this is good to go, and #255 relies on it to be committed first (they tend to conflict when I change this base)

**Review by furkansahin — APPROVED** [review]
:shipit:

**Pull request merged**